### DISCUSSION TITLE
锄大地相关玩法总结

### DISCUSSION THREAD

**Discussion by foodtooth** [LmeSzinc/StarRailCopilot#33] [Ideas]
欢迎讨论

1. 路线可用b站资源或者米游社资源，需要人力转换一次坐标
    1. 可以用带破坏物路线，还可以顺便做可能的每日和收集些合成材料
    2. 仙舟的金人从厨的稳定性上看可以考虑跳过
2. （可选）考虑可能的角色死亡，传送次回血
3. 目前看官方给的怪物追踪传送方式不稳定，可能没有比我们固定线路 + 寻路更好

**Comment by BeatSeat** [comment]
1.ii. 确实是蚊子腿，不过可以放最后打
3. 我觉得背包材料追踪可以用来辅助，比如可以知道哪张图有漏的从而考虑补怪
